client/my-sites/upgrades/domain-management/transfer/index.jsx

```diff
@@ -12,11 +12,14 @@ import VerticalNav from 'components/vertical-nav';
 import VerticalNavItem from 'components/vertical-nav/item';
 import Header from 'my-sites/upgrades/domain-management/components/header';
 import Main from 'components/main';
+import { get } from 'lodash';
 
 class Transfer extends React.Component {
 	render() {
-		const slug = this.props.selectedSite.slug,
-			domainName = this.props.selectedDomainName;
+		const slug = this.props.selectedSite.slug;
```

Comment:
So if you're tidying these up, I'd recommend a [destructuring assignment\(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) to save us a bit of typing. 

Also let's guard `selectedSite.slug`

```jsx
const { selectedSite, selectedDomainName } = selectedSite;
const slug = get( selectedSite, 'slug' );
```